cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,48 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
-
-  });
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import accountPageObject from '../support/pages/account.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  beforeEach(() => {
-
-  });
-
-  it('should provide an ability to update username', () => {
+const settingsPage = newSettingsPageObject();
+const homePage = newhomePageObject();
+const accountPage = newaccountPageObject();
```

Comment:
It seems like there is a typo in the instantiation of `accountPageObject`. It should be `new accountPageObject()` instead of `newaccountPageObject()`.